frontend/app/src/components/Sidebar/styled-components.ts

```diff
@@ -199,7 +198,9 @@ export const StyledCollapseSidebarButton =
   styled.div<StyledCollapseSidebarButtonProps>(
     ({ showSidebarCollapse, theme }) => {
       return {
-        display: showSidebarCollapse ? "inline" : "none",
+        display: "inline",
+        visibility: showSidebarCollapse ? "visible" : "hidden",
```

Comment:
We use visibility instead of display -> this allows us to just use a max width of 100% for the logo instead of the complicated calculation from before.